src/vcpkg/tools.cpp

```diff
@@ -344,6 +344,14 @@ namespace vcpkg
                 out_candidate_paths.push_back(*pf / "CMake" / "bin" / "cmake.exe");
             }
         }
+
+        virtual bool is_acceptable(const Path& exe_path) const override
+        {
+            // the cmake version from mysys and cygwin can not be used because that version can't handle 'C:' in paths
+            auto path = exe_path.generic_u8string();
+            return !Strings::ends_with(path, "/usr/bin") && !Strings::ends_with(path, "/cygwin64/bin");
```

Comment:
Is this check scoped to Windows?

Reply:
Yes by `#if defined(_WIN32)` in line 332